src/index.ts

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
this lets *nix systems know to use node js to run this script

Reply:
Nice 👍